packages/cxl-ui/src/components/cxl-card.js

```diff
@@ -0,0 +1,25 @@
+import { customElement, LitElement, html } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlCardGlobalStyles from '../styles/global/cxl-card-css.js';
+import cxlCardStyles from '../styles/cxl-card-css.js';
+
+@customElement('cxl-card')
+export class CXLCard extends LitElement {
+  static get styles() {
+    return [cxlCardStyles];
+  }
+
+  render() {
+    return html` <slot></slot> `;
+  }
```

Comment:
I'm actually a little inbetween with this essentially a simplistic wrapper element vs extending `CXLCard` also from `AccordionPanelElement` like `CXLAccordionCard`, but

* card is always open
* toggle is hidden

Reply:
I'm on board with extending `AccordionPanelElement` 👍

Reply:
> I'm on board with extending `AccordionPanelElement` +1

I'm wrapping up for today, we have this production ready as-is I think :+1: 

https://user-images.githubusercontent.com/147228/116932572-1b093b00-ac6b-11eb-99fb-8265457d2343.mp4

Food for thought: `AccordionPanelElement` directly inside `<vaadin-tab>`... feels dirty.

Maybe we just attempt this later and do more playbook stuff today.